float64[] l_arm_joint_angle
float64[] r_arm_joint_angle
float64[] l_hand_joint_angle
float64[] r_hand_joint_angle